src/main/java/frc/robot/subsystems/funnel/commandush/PowerWheels.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems.funnel.commandush;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+
+/**
+ * Add motor to PowerWheels class.
+ * Add addRequirements.
+ */
+public class PowerWheels extends CommandBase {
+    private final TalonSRX motor;
+
+    public PowerWheels(TalonSRX motor) {
+        this.motor = motor;
+        addRequirements((Subsystem) motor);
+    }
```

Comment:
This might be some crazy way of doing it that actually works and in that case I'd be impressed, but you should take the Funnel subsystem in the constructor (instead of the motor) and also add the requirements on the subsystem itself.

Reply:
You can look at your Intake code [here](https://github.com/Galaxia5987/Asteria-Offseason-2021/blob/intake/src/main/java/frc/robot/subsystems/intake/commands/Commandy.java).